projects/container_bench/testing/prepare.py

```
@@ -7,21 +7,46 @@
+
+    if config.project.get_config("cleanup.files.venv"):
+        logging.info("Cleaning up virtual environment")
+        base_work_dir = remote_access.prepare()
+        venv_path = utils.get_benchmark_script_path(base_work_dir).parent / ".venv"
+        if remote_access.exists(venv_path):
+            logging.info(f"Removing {venv_path} ...")
+            remote_access.run_with_ansible_ssh_conf(base_work_dir, f"rm -rf {venv_path}")
+
+    try:
+        cleanup_podman_platform_darwin()
+    except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)